sql/core/src/main/scala/org/apache/spark/sql/execution/command/tables.scala

```diff
@@ -1029,6 +1029,35 @@ case class ShowCreateTableCommand(table: TableIdentifier) extends RunnableComman
     builder.toString()
   }
 
+  private def showViewDataColumns(metadata: CatalogTable, builder: StringBuilder): Unit = {
+    if (metadata.schema.nonEmpty) {
+      val viewColumns = metadata.schema.map { f =>
+        val comment = f.getComment()
+          .map(escapeSingleQuotedString)
```

Comment:
Can you add tests for quoted text cases?

Reply:
test added